packages/babel-cli/src/babel/options.js

```diff
@@ -152,6 +154,10 @@ export default function parseArgv(args: Array<string>) {
 
   const errors = [];
 
+  if (commander.args.length === 0) {
+    commander.args.push("src");
```

Comment:
This is going to break https://github.com/babel/babel/blob/6b91d6434d2682ef6b0bf4ff5c9540b8ddbcd7f9/packages/babel-cli/src/babel/file.js#L228 because there will always be a filename. We'll want to allow

    babel - ...

where the `-` in place of a filename means it will read from stdin.